src/fe/implicit/_Obj.js

```diff
@@ -113,3 +112,17 @@ export const flatten = (o, prefix = '') => {
 
   return result;
 };
+
+export const entries = o => {
+  const list = [];
+
+  if (!_.isNonNullObject(o)) {
+    return list;
+  }
+
+  loop(o, (val, key) => {
```

Comment:
Will this iterate through Object's own properties?

Reply:
https://github.com/razorpay/common-frontend-utils/blob/c9540c55a0561d1a84667131a3502d82c55abf81/src/fe/implicit/_Obj.js#L30-L33